activestorage/test/dummy/config/application.rb

```diff
@@ -19,7 +19,7 @@
 
 module Dummy
   class Application < Rails::Application
-    config.load_defaults 5.1
+    config.load_defaults 5.2
```

Comment:
@georgeclaghorn I'm thinking about removing this change from this PR in favor of #30290